index.js

```diff
@@ -56,13 +57,20 @@ const init = async () => {
   // Then we load events, which will include our message and ready event.
   const evtFiles = await readdir("./events/");
   client.logger.log(`Loading a total of ${evtFiles.length} events.`);
-  evtFiles.forEach(file => {
+  await Promise.all(evtFiles.map(file => {
```

Comment:
This needs to return an array of promises. You need to add `async` in the `Array#map`'s callback.

```javascript
await Promise.all(evtFiles.map(async (file) => {
```